tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py

```diff
@@ -67,6 +86,11 @@ def tap_share_to_messages(self):
         from gaiatest.apps.messages.regions.new_message import NewMessage
         return NewMessage(self.marionette)
 
+    def open_new_window(self):
+        element = self._root_element.find_element(*self._new_window_button_locator)
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
```

Comment:
Yes, I would love to do that, but I can't import Search in this file, because it would cause a circular reference. I need to make a BaseBrowser class from which we can work from. Because then the Search app would import that class and not this file.